modules/installation-vsphere-infrastructure.adoc

```diff
@@ -89,6 +91,17 @@ you install {product-title}.
 You must ensure that the time on your ESXi hosts is synchronized before you install {product-title}. See link:https://docs.vmware.com/en/VMware-vSphere/6.7/com.vmware.vsphere.vcenterhost.doc/GUID-8756D419-A878-4AE0-9183-C6D5A91A8FB1.html[Edit Time Configuration for a Host] in the VMware documentation.
 ====
 
+[id="vsphere-csi-driver-requirements_{context}"]
+= VMware vSphere CSI Driver Operator requirements
+
+The following requirements must be met in order to install the CSI Driver Operator:
+
+* VMware vSphere version 6.7U3 or newer
+* Virtual machines of hardware version 15 or newer
+* No third party CSI driver operator already installed in the cluster
```

Comment:
Here I think we need to add "VMware ESXi version 6.7 Update 3 or greater"

Reply:
I think "VMware vSphere 6.7u3" is fine because it implies both ESXi versions and vcenter versions.